src/Router.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import Introduce from "./pages/Introduce/Introduce";
+import Product from "./pages/Product/Product";
+import ProductList from "./pages/ProductList/ProductList";
+import Review from "./pages/Product/components/Review/Review";
+import Count from "./pages/Product/components/Count/Count";
+import Color from "./pages/Product/components/Color/Color";
+
+const Router = () => {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/Introduce" element={<Introduce />} />
```

Comment:
```suggestion
        <Route path="/introduce" element={<Introduce />} />
```
경로 이름도 소문자로 시작하는 것이 좋습니다
대문자로 쓰게 되면 컴포넌트의 네이밍 컨벤션과 겹치기에 최대한 겹치지 않도록 작성해주세요!